bot/cogs/sticky.py

```diff
@@ -146,9 +146,14 @@ async def edit_sticky_message(
             else:
                 data = dict(res[0])
 
-                sticky = await channel.fetch_message(data["message_id"])
-                message = '\n'.join(message.split('\\n'))
-                await sticky.edit(content=message)
+                try:
+                    sticky = await channel.fetch_message(data["message_id"])
+                    message = '\n'.join(message.split('\\n'))
+                    await sticky.edit(content=message)
+                except discord.errors.NotFound:
+                    sticky = self.bot.get_channel(channel.id)
+                    message = '\n'.join(message.split('\\n'))
+                    await sticky.send(message)
```

Comment:
```suggestion
                try:
                    sticky_msg = await channel.fetch_message(data["message_id"])
                    message = '\n'.join(message.split('\\n'))
                    await sticky_msg.edit(content=message)
                except discord.errors.NotFound:
                    sticky_channel = self.bot.get_channel(channel.id)
                    message = '\n'.join(message.split('\\n'))
                    await sticky_channel.send(message)
```
Since it's in different return object, the variable name should be different.